### PR TITLE
Configures Packer to run Amazon-EBS Builder

### PR DESCRIPTION
**Pull request by qwwqwwq**

### CODE REVIEW

**Comment by ahsu1230** [comment]
is there a resource that you followed to do this? Mind linking it?

**Comment by qwwqwwq** [comment]
Yeah! Mostly used the docs here: https://www.packer.io/docs/builders/amazon-ebs.html

Also we use this process at work so I've done it a bunch of times before

File: config/packer/vetafi-ubuntu-14.04.4-server-amd64.json [hunk 1]
```diff
@@ -109,6 +127,11 @@
                 "provider": "virtualbox",
                 "version": "0.0.4"
             }
-        }]
+        },
+        {
+            "type": "atlas",
+            "artifact": "qwwqwwq/nodejs-mongodb-redis",
```
Comment (ahsu1230) [inline config/packer/vetafi-ubuntu-14.04.4-server-amd64.json]: I've noticed that there are a lot of personal folders in a lot of these paths. Can we make them a globally named path of sorts? This way, we can more confident things work/don't work based on personal local environments.

File: config/packer/vetafi-ubuntu-14.04.4-server-amd64.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "push": {
-      "name": "",
+      "name": "qwwqwwq/nodejs-mongodb-redis",
```
Comment (ahsu1230) [inline config/packer/vetafi-ubuntu-14.04.4-server-amd64.json]: same here

File: config/packer/vetafi-ubuntu-14.04.4-server-amd64.json (hunk 1 above)
Comment (qwwqwwq) [inline config/packer/vetafi-ubuntu-14.04.4-server-amd64.json]: This isn't a personal folder, this is the name of our image on atlas. When I signed up for atlas I used the username qwwqwwq.

**Pull request merged**